data/blog/2024/2024-06-24-archiving-api-tools.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 2024-06-24-archiving-api-tools
+author: julian
+title: 'API Tools (formerly Apigility) retirement'
+draft: false
+public: true
+created: '2024-06-24T11:00:00-01:00'
+updated: '2024-06-24T11:00:00-01:00'
+tags:
+  - meeting summary
+---
+
+On Monday, **6 May 2024**, the Technical Steering Committee for the Laminas Project held its monthly meeting.
+The topics discussed included the **API Tools (formerly Apigility)** alternatives.
+
+<!--- EXTENDED -->
+
+### Api Tools discussion
+
+The **Laminas Technical Steering Committee** decided in an initial meeting on **January 2023**, to mark
+laminas-api-tools as security only. The main reason is that laminas-api-tools main architecture is MVC and this
+architectural pattern is almost dead in favor of Middleware pattern (Mezzio). A new version of laminas-api-tools would
```

Comment:
That is not the main reason, that was one of the bases of the discussion.
This is the main reason:

> The key point brought up during the discussion is that users want us to continue maintaining it, but we do not have the resources to do so.

Reply:
clarified